extjwnl/src/main/java/net/sf/extjwnl/data/Synset.java

```diff
@@ -260,25 +260,27 @@ public boolean containsAll(Collection<?> c) {
 
         @Override
         public boolean removeAll(Collection<?> c) {
-            if (null != dictionary && dictionary.isEditable()) {
+            if (dictionary != null && dictionary.isEditable()) {
```

Comment:
It would have been better to keep existing style of "constants on the left side of operator". The original reason is an extra safeguard against `= vs ==` typos.